template.yaml

```diff
@@ -291,7 +291,6 @@ Resources:
           - "aws.s3"
         detail-type:
           - "Object Created"
-          - "Object Deleted"
```

Comment:
Our resources are "soft deleted", therefore not necessary to listen to delete events in s3

`publication.status` -> `deleted`
 `nviCandidate.applicable` -> `false`